resources/lang/en-US/mail.php

```diff
@@ -88,7 +88,7 @@
     'upcoming-audits' => 'There is :count asset that is coming up for audit within :threshold days.|There are :count assets that are coming up for audit within :threshold days.',
     'user' => 'User',
     'username' => 'Username',
-    'unaccepted_asset_reminder' => 'You have Unaccepted Assets.',
+    'unaccepted_asset_reminder' => 'You have Unaccepted Asset(s).',
```

Comment:
This weeks awkward to me - I think we'd want to trans_choice the subject here as well

Reply:
So, yea, I figured that this would catch your attention. 

I am unsure how to get that to actually work. The email text is in markdown, but the actual email title is built different. its running off a function `public function envelope(): Envelope` line 30 in UnacceptedAssetReminderMail.php

There is probably a pretty tricky way to get it to work tho...